hs/test/Tests.hs

```diff
@@ -202,21 +214,23 @@ testSpendNonexistentInput =
   let
     txbody = Tx
               (Set.fromList [TxIn genesisId 42])
-              [ TxOut aliceAddr (Coin 0) ]
+              [ TxOut aliceAddr (Coin 10000) ]
               Set.empty
+              (Coin 1500)
     aliceWit = makeWitness alicePay txbody
     tx = TxWits txbody (Set.fromList [aliceWit])
-  in ledgerState [tx] @?= Left [BadInputs, InsufficientWitnesses]
+  in ledgerState [tx] @?= Left [BadInputs, ValueNotConserved, InsufficientWitnesses]
```

Comment:
This changes the semantics of the test case. Shouldn't this be at least in a separate commit?

Reply:
changing the preservation of value from and inequality to an equality changes the semantics. it causes some correlation between the errors that is different than how it was before. I'm not sure how to make the equality change without also making a test change (or have tests broken between commits).  Or am I missing something?

Reply:
you are perfeclty right. For me, it wouldn't be bad to have a test failing in one commit if it's fixed within the next commit. But that might be problematic for bisection maybe? So leave it like it is if you want to.